pyproject.toml

```
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["flit_core >=2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.metadata]
+author = "Ciencia de Datos • GECI"
+author-email = "ciencia.datos@islas.org.mx"
+classifiers = [ "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",]
+description-file = "README.md"
+home-page = "https://github.com/IslasGECI/clean-enoa.py"
```

¿Crees que convenga renombrar este repo a `clean_enoa` en lugar de `clean-enoa`? 

```suggestion
home-page = "https://github.com/nepito/clean_enoa.py"
```

---

¿Por qué el cambio de nombre? ¿Cuáles pueden ser las ventajas o desventajas de uno u otro?

---

@davidmacer El URL del repo dice `clean-enoa` con guión medio, pero la carpeta donde se encuentra el módulo se llama ` clean_enoa` con piso. Mi propuesta es que el URL coincida con el nombre de la carpeta. La ventaja es consistencia. La desventaja podría ser un cambio innecesario.


Además, sobraba un `.py` que heredamos de un copipasteo, pero eso ya quedó.

---

Listo @evaristor . Cambié el nombre del repositorio. Primero intenté cambiar el nombre del módulo a `clean-enoa` pero el `tox` me marcó error. Parece que no le gustan nombres para módulos con guiones.

---

@nepito Creo que algunos lenguajes tienen problemas con los guines porque intentan hacer una resta